### PR TITLE
feat: show recovery codes

### PR DESCRIPTION
**Pull request by leMaur**
## Summary
https://app.clickup.com/t/gb2r10

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: resources/views/profile/two-factor-authentication-form.blade.php [hunk 1]
```diff
@@ -91,7 +91,8 @@ class="button-secondary w-full sm:w-auto"
             <span class="header-4">@lang('fortify::pages.user-settings.2fa_title')</span>
             <span class="mt-4">@lang('fortify::pages.user-settings.2fa_description')</span>
 
-            <div class="flex mt-8">
+            <div class="mt-4 sm:mt-8 flex flex-col items-center space-y-4 sm:items-start sm:flex-row sm:space-y-0 sm:space-x-6">
+                @svg('fortify-profile.2fa', 'w-48')
```
Comment (ItsANameToo) [inline resources/views/profile/two-factor-authentication-form.blade.php]: `x-ark-icon`

**Review by ItsANameToo — CHANGES_REQUESTED** [review]
- the image is not centered in the modal
image
- the current images have blue highlights, which will be MSQ specific so they should not end up in fortify

File: resources/views/profile/two-factor-authentication-form.blade.php [hunk 2]
```diff
@@ -91,7 +91,8 @@ class="button-secondary w-full sm:w-auto"
             <span class="header-4">@lang('fortify::pages.user-settings.2fa_title')</span>
             <span class="mt-4">@lang('fortify::pages.user-settings.2fa_description')</span>
 
-            <div class="flex mt-8">
+            <div class="mt-4 sm:mt-8 flex flex-col items-center space-y-4 sm:items-start sm:flex-row sm:space-y-0 sm:space-x-6">
+                <img src="{{ asset('/images/profile/2fa.svg') }}" class="w-24 h-24" alt="">
```
Comment (leMaur) [inline resources/views/profile/two-factor-authentication-form.blade.php:95]: if I use `x-ark-icon` the image doesn't load..

File: resources/views/profile/two-factor-authentication-form.blade.php [hunk 3]
```diff
@@ -167,25 +172,39 @@ class="w-full justify-center"
...
+            <x-slot name="title">
+                @lang('fortify::forms.confirm-password.title')
+            </x-slot>
 
-        @endif
-    </div> --}}
+            <x-slot name="description">
+                <div class="flex flex-col mt-4">
+                    <div class="flex justify-center w-full mt-8">
+                        <img src="{{ asset('/images/auth/confirm-password.svg') }}" class="w-auto h-auto" alt="">
```
Comment (leMaur) [inline resources/views/profile/two-factor-authentication-form.blade.php:184]: same here..

File: resources/views/profile/two-factor-authentication-form.blade.php (hunk 2 above)
Comment (ItsANameToo) [reply resources/views/profile/two-factor-authentication-form.blade.php:95]: yeah for the `img` ones you can leave it, it's purely when you use `@svg` it's better to use `x-ark-icon` instead. The `img` tags are usually used when it's for images that we need to be able to overwrite or are expected to exist in the project but are not part of fortify itself

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**